src/pages/Signup/styles.scss

```diff
@@ -1,5 +1,5 @@
 @import '../../utils/variables.scss';
-@import '../Login/LoginPage.scss';
+@import '../Login/styles.scss';
```

Comment:
why are we using login styles here in signup?

Reply:
The two pages are essentially the same, kuya, except for the Signup Card component so I decided to reuse the Login style sheet nalang. Should I change this? ;-;